reset-test-data.sh

```diff
@@ -33,10 +33,10 @@ if [ $LOCAL == "1" ]; then
   export $(grep -v '^#' .env | xargs)
 fi
 
-if [ $LOCAL == "1" ]; then
-  echo -e "Getting AWS credentials ..."
-  eval "$(gds aws ${GDS_AWS_ACCOUNT} -e)"
-  echo "done!"
-fi
+#if [ $LOCAL == "1" ]; then
```

Comment:
I think you're going to have to leave these lines in uncommented and when you run locally comment them out temporarily locally for your use, but uncomment them again before pushing up. The tests don't work for me with those lines commented out. I'll find out if there is a solution that works for both of us but in the mean time we'll (you'll) just have to take the pain I'm afraid.